lib/srv/db/server_test.go

```diff
@@ -68,13 +70,21 @@ func TestDatabaseServerStart(t *testing.T) {
 	}
 
 	// Make sure servers were announced and their labels updated.
-	servers, err := testCtx.authClient.GetDatabaseServers(ctx, apidefaults.Namespace)
-	require.NoError(t, err)
-	require.Len(t, servers, 4)
-	for _, server := range servers {
-		require.Equal(t, map[string]string{"echo": "test"},
-			server.GetDatabase().GetAllLabels())
-	}
+	retryutils.RetryStaticFor(5*time.Second, 20*time.Millisecond, func() error {
```

Comment:
Any particular reason to use `retryutils` in test code over `EventuallyWithT`?

Reply:
yeah EventuallyWithT always sleeps, this almost always passes without sleeping

Reply:
RetryStaticFor is "intended mostly for tests" https://github.com/gravitational/teleport/blob/5292b2bfee0c5cd8f6ab7586c03a3737de2303c4/api/utils/retryutils/retry.go#L221